nengo_viz/static/viz_sim_control.js

```diff
@@ -182,23 +173,6 @@ VIZ.TimeSlider = function(args) {
     /** scale to convert time to x value (in pixels) */
     this.kept_scale = d3.scale.linear();
     
-    /** create the overall div */
-    this.div = document.createElement('div');
```

Comment:
No, I mean did it used to be a bug? With the old code, could we have ended up with lots of divs if we resized a lot?